snuba/query/processors/timeseries_processor.py

```diff
@@ -57,11 +57,23 @@ def __init__(
                                 [
                                     String("toStartOfHour"),
                                     String("toStartOfMinute"),
+                                    String("toStartOfDay"),
                                     String("toDate"),
                                 ]
                             ),
                             (column_match, LiteralMatch(Any(str))),
```

Comment:
Do we need to match these two cases (this one and the one you added) explicitly? Isn't the one below + optional parameters enough ?

Reply:
You're right, just the one is fine.